typeql/reasoner/attribute-attachment.feature

```diff
@@ -18,16 +18,8 @@
 #noinspection CucumberUndefinedStep
 Feature: Attribute Attachment Resolution
 
-  Background: Set up databases for resolution testing
-    Given connection has been opened
-    Given connection does not have any database
-    Given connection create database: reasoned
-    Given connection create database: materialised
-    Given connection open schema sessions for databases:
-      | reasoned     |
-      | materialised |
-    Given for each session, open transactions of type: write
-    Given for each session, typeql define
+  Background: Set up database
+    Given schema
```

Comment:
with the current naming convention, I'd expect to use these steps in ALL of our BDD, not just in the resolution steps. However, the implementation is just for resolution -- I think what we should do is call this `Given resolution schema`, `Given resolution data`, and `Given resolution query`

Reply:
Gone with `Given reasoner schema`, `Given reasoner data` and `Given reasoner query`